angular.js

```diff
@@ -32,5 +32,28 @@ module.exports = {
             Overrides to Angular extra recommended rules:
             https://github.com/angular-eslint/angular-eslint/blob/master/packages/eslint-plugin/src/configs/recommended--extra.json
         */
+
+        /*
+            Overrides to Angular rules outside of the recommended configuration:
+        */
+
+        '@angular-eslint/component-max-inline-declarations': 'error',
+        '@angular-eslint/component-selector': 'error',
+        '@angular-eslint/contextual-decorator': 'error',
+        '@angular-eslint/directive-selector': 'error',
+        '@angular-eslint/no-attribute-decorator': 'error',
+        '@angular-eslint/no-forward-ref': 'error',
+        '@angular-eslint/no-input-prefix': 'error',
+        '@angular-eslint/no-lifecycle-call': 'error',
+        '@angular-eslint/no-pipe-impure': 'error',
+        '@angular-eslint/no-queries-metadata-property': 'error',
+        '@angular-eslint/pipe-prefix': 'error',
+        '@angular-eslint/prefer-on-push-component-change-detection': 'error',
+        '@angular-eslint/prefer-output-readonly': 'error',
+        '@angular-eslint/relative-url-prefix': 'error',
+        '@angular-eslint/sort-ngmodule-metadata-arrays': 'error',
+        '@angular-eslint/use-component-selector': 'error',
+        '@angular-eslint/use-component-view-encapsulation': 'error',
```

Comment:
[@angular-eslint/use-component-view-encapsulation](http://codelyzer.com/rules/use-component-view-encapsulation)

Reply:
This rule is interesting. You have explicitly disable view encapsulation, so it's almost certainly intentional. I don't know if this is for environments where they're having trouble with people disabling encapsulation?

Reply:
One nice thing about causing it to be an error is that it forces devs to add a comment suppressing the rule, which helps highlight that a component has diverged from the normal encapsulation behavior. And if your codebase has a policy of requiring an explanation when suppressing rules, then that explanation can serve as documentation for why it's diverging.

Anyway, I agree to keep it an error, just speculating.

Reply:
@rajsite This thread on @angular-eslint/use-component-view-encapsulation is awaiting your vote.

Reply:
@TrevorKarjanis this is ready to close.